x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -26,8 +30,82 @@ import (
 
 const debugSelector = "synthexec"
 
+// ensure compatability of synthetics by enforcing the installed
+// version never goes beyond this range
+const ExpectedSynthVersion = "<2.0.0"
+
+type PackageJSON struct {
```

Comment:
I'd use lowercase since we don't need to export this. We aren't 100% consistent across our codebase here, but where we catch it's good to default to private.